.github/workflows/e2e-test.yaml

```diff
@@ -46,23 +46,30 @@ jobs:
         run: |
           curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
           source /home/runner/.profile
-          tiup update --nightly
-          tiup playground nightly --tiflash=0 &
+          tiup install tidb:v4.0.5 tikv pd prometheus
```

Comment:
It can be simply

```
tiup playground v4.0.5 --tiflash=0
```

Reply:
It may block the jobs, let me have a try.

Reply:
Right. Maybe we need something to wait TiDB started successfully, like https://github.com/tikv/copr-test/blob/master/push-down-test/run-tests.sh#L283

Reply:
I write a shell to wait for it by check the` tiup playground display` command, but the tiup 1.1.1 version has a bug that `tiup playground display` doesn't work normally... will use it when tiup release 1.1.2